### PR TITLE
Added new ad network: Nativery

### PR DESCRIPTION
**Pull request by acardinale**
Hi Team,
we have added our ad network "Nativery".

- Add network logic
- Add samples and doc
- Update configuration files

Could you review our commits?

Regards.

### CODE REVIEW

**Comment by lannka** [comment]
@powerivq to take a look

File: ads/nativery.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function nativery(global, data) {
+  validateData(data, ['wid']);
+  const params = Object.assign({}, data);
+
+  // push the two object into the '_nativery' global
+  window._nativery =
```
Comment (powerivq) [inline ads/nativery.js]: Curious, why not using `global`?

File: ads/nativery.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        wid: data.wid,
+        referrer: data.referrer || window.context.referrer,
+        url: data.url || window.context.canonicalUrl,
+        viewId: window.context.pageViewId,
+        params,
+      }
+    );
+
+  // must add listener for resize
+  window.addEventListener('amp-widgetCreated', function(e) {
```
Comment (powerivq) [inline ads/nativery.js]: Not sure if I have seen this event. What is it meant for?

File: ads/nativery.js (hunk 1 above)
Comment (acardinale) [reply ads/nativery.js]: Hi @powerivq, I was inspired by another ad network already present in amp.
I changed it with new commit

File: ads/nativery.js (hunk 2 above)
Comment (acardinale) [reply ads/nativery.js]: Hi @powerivq, It's a custom event to adjust the height based on the content

**Comment by powerivq** [comment]
Approved. Pls fix the lint error tho. Lmk when you wanna launch.

**Comment by acardinale** [comment]
@powerivq I did it

**Comment by powerivq** [comment]
@jridgewell  for approval.

**Comment by acardinale** [comment]
> @jridgewell for approval.

@jridgewell Would you review, please?

**Pull request merged**